scripts421.sql

```diff
@@ -0,0 +1,14 @@
+
+ALTER TABLE students ADD CONSTRAINT min_age CHECK ( age >= 16 );
+ALTER TABLE students ALTER COLUMN name SET NOT NULL ;
+ALTER TABLE students ADD CONSTRAINT unique_name UNIQUE(name);
+/* But the next one will be redundant:  */
```

Comment:
why?

Reply:
HERE: Because we already have the CONSTRAINT min_age, which will oblige assigning some appropriate value for age